src/NuGetPackageVerifier/Program.cs

```diff
@@ -213,8 +217,6 @@ public static int Main(string[] args)
 
                 foreach (var unprocessedPackage in unprocessedPackages)
                 {
-                    logger.LogWarning("\tUnprocessed package: {0} ({1})", unprocessedPackage.Id, unprocessedPackage.Version);
```

Comment:
Coherence-Signed is full of these messages since we don't list any of the packages there.:

`EXEC Unprocessed package: Microsoft.AspNetCore.Authentication.Facebook (1.1.0-alpha1-21491)`

Doesn't seem super helpful having this around anymore. Let me know if you think otherwise.